src/commands/jobs/util.ts

```diff
@@ -263,9 +263,9 @@ export const itemOutputFormat = {
 				if (!val) {
 					return '';
 				}
-				const size = !Boolean(val.source_to_destination)
-					? (val.source && val.source.bytes)
-					: (val.destination && val.destination.bytes);
+				const size = Boolean(val.source_to_destination) && val.destination && val.destination.bytes
+					? (val.destination && val.destination.bytes)
+					: (val.source && val.source.bytes);
```

Comment:
does this change make sense or should i create a different output format for folder size? my concern with this logic is that if an item has 0 bytes on the destination, but it's not 0 bytes on the source, you'd see the source size, which may give a false impression of the contents on the destination...if there was a problem with the transfer.